osu.Game.Rulesets.Taiko/UI/DrumTouchInputArea.cs

```diff
@@ -1,21 +1,26 @@
 // Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
 // See the LICENCE file in the repository root for full licence text.
 
+#pragma warning disable IDE0001 // Simplify Names
```

Comment:
This doesn't seem to be suppressing any warnings for me locally, but regardless shouldn't need to be used (best to follow formatting rules unless actually required).

Reply:
For me, it kept aggressively reminding me to "simplify" every instance of "TaikoAction" to "TaikoInput" once I registered TaikoInput